utils/cups-browsed.c

```diff
@@ -694,6 +697,27 @@ stop_debug_logging()
   lfp = NULL;
 }
 
+// returns the size of debug log file
+long int findLogFileSize() 
+{ 
+    FILE* fp = fopen(debug_log_file, "r"); 
+    if (fp == NULL) { 
+        return -1; 
+    } 
+    fseek(fp, 0L, SEEK_END); 
+    long int res = ftell(fp); 
+    fclose(fp); 
+    return res; 
+}
+
+void copyToFile(FILE **fp1, FILE **fp2){
+  char ch;
+  while((ch = getc(*fp1)) != EOF)
+    putc(ch, *fp2);
+  fclose(*fp1);
```

Comment:
Hi @dipanshu231099 ,

I'm Zdenek and I'm Fedora/CentOS printing stack maintainer and I sometimes check the pull request in cups-filters to have an other pair of eyes on the merge and help Till with it.
Would it be possible to move those ```fclose()``` calls into the function where you call ```fopen()```? IMO it is easier to read whether the file descriptor is closed or not.

Reply:
Yes sure.

Reply:
Solved in https://github.com/OpenPrinting/cups-filters/pull/267/commits/d8e30a81bac7bcdfc8d1399bf204328d36ae866e.